app/konnect/dev-portal/applications/external-client-providers/okta.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Configuring Okta for Dynamic Client Registration
+no_version: true
+toc: true
+content-type: how-to
+---
+
+
+## Prerequisites
+
+To use Okta as an external client provider for Dynamic Client Registration (DCR), create or use an existing Okta account [here](http://developer.okta.com). Once you have an account, you will need to create a token, find the Issuer URL, and optionally set any scopes or claims.
+
+### Issuer URL
+
+1. Select **Security** from the menu.
+
+2. Select **Security > API**. The default Issuer URL should be displayed in the Authorization Servers tab. If you are using an Authorization Server that you configured, copy the Issuer URL for that Authorization Server.
+
+### Creating a token
+
+1. Select **Security** from the menu.
+
+2. Select **Security > API**.
+
+3. Click the **Tokens** tab.
+
+4. Click the **Create token** button.
+
+5. Enter a name for your token, and then copy the token value. Make sure you store this in a place you can reference, as it will only be visible as a hash afterwards.
+
+### Scopes & Claims
+
+1. Select **Security** from the menu.
+
+2. Select **Security > API**.
+
+3. Select the Authorization Server that you want to configure.
+
+4. If you would like to create an additional scope, select the **Scopes** tab, and then click the **Add Scope** button.
+
+   {:.note}
+   > Note that you can use any of the existing scopes besides **openid**, as openid
+   scope will prevent you from using client credentials.
+
+5. In order to map an application from the Dev Portal to Okta, we need to create a claim. Select the **Claims** tab. and then click the **Add Claim** button.
+
+6. Enter a name for this claim, and enter **app.clientId** for value. We can leave the Value type as expression, and include it in any scope.
+
+## Configuring the Dev Portal
+
+Once you have Okta configured, we can now setup the Dev Portal to use Okta for DCR.
+
+1. Sign in to {{site.konnect_short_name}}, and select **Dev Portal** from the menu.
+
+2. Click **Settings** to open your Dev Portal settings.
+
+3. Click the **Application Setup** tab to open the DCR settings for your portal.
+
+4. Enter the **Issuer URL** for your authorization server, and the **Token** that you created in Okta.
+
+5. Enter the names of the **Scopes** and **Claims** as comma-separated values in their corresponding inputs.
```

Comment:
I think we should mention the scope needing to not be openid (which will be default if nothing is entered) here instead of above. 

We should also mention the claim should match up to the one set up in the above section